app/assets/stylesheets/application.scss

```
@@ -1,3 +1,9 @@
+$govuk-assets-path: "/emcs/create-movement/assets/lib/govuk-frontend/govuk/assets/";
+$hmrc-assets-path: "/emcs/create-movement/assets/lib/hmrc-frontend/hmrc/assets/";
```

The path needs to change to:
```suggestion
$govuk-assets-path: "/emcs/alert-or-rejection/assets/lib/govuk-frontend/govuk/assets/";
$hmrc-assets-path: "/emcs/alert-or-rejection/assets/lib/hmrc-frontend/hmrc/assets/";
```

---

(the assets calls return 404)